modules/apps/headless-apio/forms/forms-apio-client-test/src/testIntegration/java/com/liferay/forms/apio/client/test/SelectFormApioTest.java

```diff
@@ -0,0 +1,266 @@
+			"Accept", "application/hal+json"
+		).header(
+			"Accept-Language", "en-US"
+		).when(
+		).get(
+			_formHrefURL.toExternalForm() + "?embedded=structure"
+		).then(
+		).log(
+		).ifError(
+		).statusCode(
+			200
+		).root(
+			"_embedded.formPages._embedded[0]._embedded.fields._embedded." +
+				"find {it.name == '%s'}",
+			withArgs(SelectFieldFormApioTestBundleActivator.SELECT_FIELD_NAME)
+		).body(
+			"hasFormRules", isBoolean()
+		).body(
+			"required", isBoolean()
+		).body(
+			"showLabel", isBoolean()
+		);
+	}
+
+	@Test
+	public void testSelectFieldsDataTypeIsDisplayed() {
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).header(
+			"Accept-Language", "en-US"
+		).when(
+		).get(
+			_formHrefURL.toExternalForm() + "?embedded=structure"
+		).then(
+		).log(
+		).ifError(
+		).statusCode(
+			200
+		).root(
+			"_embedded.formPages._embedded[0]._embedded.fields._embedded." +
+				"find {it.name == '%s'}",
+			withArgs(SelectFieldFormApioTestBundleActivator.SELECT_FIELD_NAME)
+		).body(
+			"dataType", equalTo("string")
+		);
+	}
+
+	@Test
+	public void testSelectFieldsInputControlIsDisplayed() {
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).header(
+			"Accept-Language", "en-US"
+		).when(
+		).get(
+			_formHrefURL.toExternalForm() + "?embedded=structure"
+		).then(
+		).log(
+		).ifError(
+		).statusCode(
+			200
+		).root(
+			"_embedded.formPages._embedded[0]._embedded.fields._embedded." +
+				"find {it.name == '%s'}",
+			withArgs(SelectFieldFormApioTestBundleActivator.SELECT_FIELD_NAME)
+		).body(
+			"inputControl", equalTo("select")
+		);
+	}
+
+	@Test
+	public void testSelectFieldsLabelIsDisplayed() {
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).header(
+			"Accept-Language", "en-US"
+		).when(
+		).get(
+			_formHrefURL.toExternalForm() + "?embedded=structure"
+		).then(
+		).log(
+		).ifError(
+		).statusCode(
+			200
+		).root(
+			"_embedded.formPages._embedded[0]._embedded.fields._embedded." +
+				"find {it.name == '%s'}",
+			withArgs(SelectFieldFormApioTestBundleActivator.SELECT_FIELD_NAME)
+		).body(
+			"label", equalTo("My Select Field")
+		);
+	}
+
+	@Test
+	public void testSelectFieldsMultipleIsDisplayed() {
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).header(
+			"Accept-Language", "en-US"
+		).when(
+		).get(
+			_formHrefURL.toExternalForm() + "?embedded=structure"
+		).then(
+		).log(
+		).ifError(
+		).statusCode(
+			200
+		).root(
+			"_embedded.formPages._embedded[0]._embedded.fields._embedded." +
+				"find {it.name == '%s'}",
+			withArgs(
+				SelectFieldFormApioTestBundleActivator.
+					MULTIPLE_SELECT_FIELD_NAME)
+		).body(
+			"multiple", equalTo(true)
```

Comment:
Please check the other field which has the property multiple equals false.

Reply:
Add check multiple equals false for other field. Ty.